src/instructlab/model/evaluate.py

```diff
@@ -248,7 +248,8 @@ def launch_server(
 
     eval_serve = copy(ctx.obj.config.serve)
     if backend is None:
-        eval_serve.backend = backends.VLLM
+        backend = eval_serve.backend = backends.VLLM
```

Comment:
Just to make sure I'm understanding this correctly - this is setting `backend` and `eval_serve.backend` vars to the value of `backends.VLLM`?

Reply:
Correct

Reply:
Update, I've changed the logic to figure out the backend with backends.  That resolves the other issue of potentially setting up params for the wrong type.